README.md

```diff
@@ -97,67 +113,93 @@ datadog.addLambdaFunctions([<LAMBDA_FUNCTIONS>])
+    jsii.String("Datadog"),
+    &ddcdkconstruct.DatadogProps{
+        NodeLayerVersion:      jsii.Number(<LAYER_VERSION>),
+        AddLayers:             jsii.Bool(<BOOLEAN>),
+        Site:                  jsii.String(<SITE>),
+        ApiKey:                jsii.String(os.Getenv("DD_API_KEY")),
+        // ...
+    })
+datadog.AddLambdaFunctions(&[]interface{}{myFunction}, nil)
+datadog.AddForwarderToNonLambdaLogGroups()
+```
+
 ## Source Code Integration
+
 [Source code integration](https://docs.datadoghq.com/integrations/guide/source-code-integration/) is enabled by default through automatic lambda tagging, and will work if:
 
-- The Datadog Github Integration is installed.
+- The Datadog Github integration is installed.
 - Your datadog-cdk dependency satisfies either of the below versions:
   - `datadog-cdk-constructs-v2` >= 1.4.0
   - `datadog-cdk-constructs` >= 0.8.5
 
 ### Alternative Methods to Enable Source Code Integration
-If the automatic implementation doesn't work for your case, please follow one of the two guides below. 
+
+If the automatic implementation doesn't work for your case, please follow one of the two guides below.
 
 **Note: these alternate guides only work for Typescript.**
+
 <details>
-  <summary>datadog-cdk version satisfied, but Datadog Github Integration NOT installed</summary>
+  <summary>datadog-cdk version satisfied, but Datadog Github integration NOT installed</summary>
 
-  If the Datadog Github Integration is not installed, you need to import the `datadog-ci` package and manually upload your Git metadata to Datadog.
-  We recommend you do this where your CDK Stack is initialized.
+If the Datadog Github integration is not installed, you need to import the `datadog-ci` package and manually upload your Git metadata to Datadog.
+For the best results, import the `datadog-ci` package where your CDK Stack is initialized.
 
-  ```typescript
-  const app = new cdk.App();
+```typescript
+const app = new cdk.App();
 
-  // Make sure to add @datadog/datadog-ci via your package manager
-  const datadogCi = require("@datadog/datadog-ci");
-  // Manually uploading Git metadata to Datadog.
-  datadogCi.gitMetadata.uploadGitCommitHash('{Datadog_API_Key}', '<SITE>')
+// Make sure to add @datadog/datadog-ci via your package manager
+const datadogCi = require("@datadog/datadog-ci");
+// Manually uploading Git metadata to Datadog.
+datadogCi.gitMetadata.uploadGitCommitHash("{Datadog_API_Key}", "<SITE>");
 
-  const app = new cdk.App();
-  new ExampleStack(app, "ExampleStack", {});
+const app = new cdk.App();
+new ExampleStack(app, "ExampleStack", {});
+
+app.synth();
+```
 
-  app.synth();
-  ```
 </details>
 <details>
   <summary>datadog-cdk version NOT satisfied</summary>
 
-  Change your initialization function as follows (note: we're changing this to pass just the `gitHash` value to the CDK):
+Change your initialization function as follows (note: we're changing this to pass just the `gitHash` value to the CDK):
```

Comment:
```suggestion
Change your initialization function as follows (**Note:** We're changing this to pass just the `gitHash` value to the CDK):
```
Can you rephrase the note to not use "we?" I'm not sure if replacing it with "Datadog" is appropriate here, so I didn't make a wording suggestion.

Reply:
I agree that "we" is confusing. Let me change it to:
Change your initialization function as follows (in this case, `gitHash` value is passed to the CDK):